testing/ceph/APKBUILD

```diff
@@ -0,0 +1,554 @@
+# Contributor: John Coyle <dx9err@gmail.com>
+# Maintainer: John Coyle <dx9err@gmail.com>
+
+: ${WITH_TESTS:=OFF}
+
+pkgname=ceph
+pkgver=11.0.2
+pkgrel=0
+pkgdesc="User space components of the Ceph file system"
+pkgusers="ceph"
+pkggroups="ceph"
+url="http://ceph.com"
+arch="all"
+license="LGPL-2.1 and CC-BY-SA-1.0 and GPL-2.0 and BSL-1.0 and GPL-2.0-with-autoconf-exception and BSD-3-Clause and MIT"
+depends="
+	ceph-osd
+	ceph-mds
+	ceph-mon
+	binutils
+"
+
+makedepends="
+	autoconf
+	automake
+	acl-dev
+	argp-standalone
+	binutils
+	boost-dev
+	btrfs-progs
+	build-base
+	bzip2-dev
+	ccache
+	cmake
+	coreutils
+	cryptsetup
+	curl-dev
+	cython
+	cython-dev
+	diffutils
+	eudev-dev
+	expat-dev
+	fcgi-dev
+	findutils
+	flex
+	fuse-dev
+	git
+	grep
+	jq
+	keyutils-dev
+	leveldb-dev
+	libaio-dev
+	libatomic_ops-dev
+	libedit-dev
+	libressl-dev
+	libtirpc-dev
+	libtool
+	libxml2-dev
+	linux-headers
+	lvm2-dev
+	nss-dev
+	openldap-dev
+	parted
+	procps
+	python-dev
+	py-pip
+	py-nose
+	py-sphinx
+	py-virtualenv
+	readline-dev
+	rpcgen
+	sed
+	snappy-dev
+	util-linux
+	userspace-rcu-dev
+	xfsprogs-dev
+	xmlstarlet
+	yasm
+"
+
+case $pkgrel in
+	''|*[!0-9]*)
+		_builddir="$srcdir"/$pkgname-$pkgver-`echo $pkgrel | sed 's/\./-/'`
+		_sourcepkg=ceph-$pkgver-`echo $pkgrel | sed 's/\./-/'`.tar.bz2
+		;;
+	*)
+		_builddir="$srcdir"/$pkgname-$pkgver
+		_sourcepkg="http://github.com/dx9/ceph/releases/download/$pkgver/$pkgname-$pkgver.tar.bz2"
```

Comment:
not yet. was still waiting on a few merges. it's ready to move to official release with upcoming 11.1.0